docs/docsearch.js

```diff
@@ -0,0 +1,85 @@
+$(function() {
```

Comment:
Whoa cool

Reply:
ha yeah! this just came with `pkgdown::build_site` on the newest version